onnx/defs/nn/defs.cc

```diff
@@ -1829,6 +1829,33 @@ bias. The formula of this operator is: If x < -lambd, y = x + bias;
 If x > lambd, y = x - bias; Otherwise, y = 0.
 )DOC";
 
+static float shrink_default_lambd = 0.5f;
+static float shrink_default_bias = 0.0f;
+
+bool BuildContextDependentFunctionBodyShrink(
+    const FunctionBodyBuildContext& ctx,
+    const OpSchema& schema,
+    FunctionProto& functionProto) {
+  float lambd = ctx.getAttribute("lambd") != nullptr ? ctx.getAttribute("lambd")->f() : shrink_default_lambd;
```

Comment:
One general question / comment: I think, in the long term, it would be better to move towards simple function (instead of context-dependent function). I think we could do that by using attribute-references for such attributes.

The main issue, I think, is the use of default-values for these attributes. I think it may be a good idea to make it the responsibility of the function-inliner to use the default-values when inlining a call where the attribute is not explicitly provided.

Does that make sense? I realize that the existing implementation of inliner will need to be updated to do this, but just want to discuss the idea first.

Reply:
+1 on moving towards to simple function.